otx/algorithms/segmentation/configs/base/models/semi_mt_segmentor.py

```diff
@@ -0,0 +1,10 @@
+"""Segmentor config for semi-supervised learning."""
```

Comment:
How about making `semisl` folder inside of `otx/algorithms/segmentation/configs/base/models`?
I can't imagine the meaning of `mt`  in this case. 

If don't want to make `semisl` folder, changing name of this file is needed for easy understanding. 
(i.e. Since this python file was already located in the `segmentation` folder, we can delete segmentor for this case. `semi_mt_segmentor.py` --> `semi_mean_teacher.py` or `mean_teacher.py`

Reply:
I agree with sungman on this. But rather than creating a separate folder, just mean_teacher.py looks better.